connaisseur/exceptions.py

```diff
@@ -57,6 +61,14 @@ class AmbiguousDigestError(BaseConnaisseurException):
     pass
 
 
+class CosignError(BaseConnaisseurException):
+    pass
+
+
+class CosignTimeout(BaseConnaisseurException):
+    pass
```

Comment:
Maybe create an exception hierarchy, with `BaseConaisseurException` --> `CosignException` --> `CosignError` etc., to avoid polluting the namespace once several attestation mechanisms are supported.

Speaking of disentangling things, it may also be worth thinking about whether stuff specific to an attestation method should live inside a subdirectory (here: `cosign/exceptions.py`, `cosign/validate.py` etc., similar for Notary v1/v2).

Reply:
Maybe we should move this into the overall refactoring: https://github.com/sse-secure-systems/connaisseur/pull/101. Like keep it flat for now and then choose a proper solution with the central refactoring? Otherwise, I would feel that part of the code base is going ahead in one of the directions and it will be changed later anyways...

Reply:
@phbelitz what do you think? Should I delay this until the refactoring is done to consider multi-validator support?

Reply:
if this is merged before the refactoring, then yes keep it flat and i will make things consistent in the refactoring PR